src/styles/createBreakpoints.spec.js

```diff
@@ -21,8 +21,20 @@ describe('createBreakpoints', () => {
   });
 
   describe('down', () => {
-    it('should work', () => {
-      assert.strictEqual(breakpoints.down('md'), '@media (max-width:959.95px)');
+    it('should work for xs', () => {
```

Comment:
Using `sm` instead of `xs` would have allowed showing that the assertion don't change.